frontend/src/bundles/common/components/bar-chart/bar.module.css

```diff
@@ -0,0 +1,4 @@
+.chart {
+    height: 400px;
```

Comment:
I am not sure about height

Reply:
fixed: [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/32/commits/1203288c50575a8d15e088825517d0b2f0fbb67b](url)